templates/pages/dashboard.html

```diff
@@ -26,19 +26,38 @@
 </div>
 {% endif %}
 
-<p>
-  <a class="btn btn-primary" href="/submit">
-    <i class="fa fa-pen"></i>
-    Add Post
-  </a>
-  <a class="btn btn-secondary" href="/import">
-    <i class="fas fa-file-csv"></i>
-    Import
-  </a>
-  <a class="btn btn-secondary" href="/export">
-    <i class="fas fa-file-csv"></i>
-    Export
-  </a>
-</p>
+<div class="row no-gutters">
+  <div class="col-md col-6">
+    <a class="btn btn-primary btn-block" href="/submit">
+      <i class="fa fa-pen"></i>
+      Add Post
+    </a>
+  </div>
+  <div class="col-md col-6">
+    <a class="btn btn-secondary btn-block" href="/import">
+      <i class="fas fa-file-csv"></i>
+      Import
+    </a>
+  </div>
+  <div class="col-md col-6">
+    <a class="btn btn-secondary btn-block" href="/export">
+      <i class="fas fa-file-csv"></i>
+      Export
+    </a>
+  </div>
+  <div class="col-md col-6">
+    <div class="dropdown show">
+      <a class="btn btn-secondary btn-block dropdown-toggle" href="#" role="button" id="viewSelect" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        View
+      </a>
+      <div class="dropdown-menu" aria-labelledby="viewSelect">
```

Comment:
I think it makes sense to right-align the dropdown here:
```suggestion
      <div class="dropdown-menu dropdown-menu-right" aria-labelledby="viewSelect">
```